docs/source/gbd2019_models/causes/maternal_disorders/index.rst

```diff
@@ -230,7 +232,23 @@ Ratios of maternal disorder mortality and incidence are defined in the table bel
      - 
    * - Incident maternal disorders
      - incidence_rate_c366 / (incidence_p * prevalence_np)
-     - **Post-maternal disorder state persists for one timestep**
+     - **Post-maternal disorder state persists for one timestep** (one week, as implemented).
+
+.. note::
+
+  GBD incident cases of the maternal disorders parent cause is equal to the *sum* of the incident cases of each maternal disorders sub-causes rather than a count of individuals who experience at least one maternal disorder cause during gestation/birth/postpartum. Therefore, for some demographic groups with high maternal disorders burdens (particularly older age groups), it is possible that the above ratio is greater than one.
+
+  As currently implemented, the incident maternal disorders ratio per birth was not clipped at 1 in the input data and was used as-is in the simulation as it did not violate Vivarium processes to have values above one for this parameter.
+
+  Notably, this strategy has important implications:
+
+    - If there are particular demographic groups that have incident maternal disorders ratios per birth greater than one, then we will underestimate total incident maternal disorder cases (and associated YLDs) for that group in our simulation. For the IV iron simulation, while we underestimated incident maternal disorders cases for high risk groups (older ages), this did not have a large impact on the popuation-level incident maternal disorder cases because fertility rates were low in these groups.
+
+    - Our modeling strategy assumes that maternal disorders cases and YLDs are spread widely across the population of PLW rather than being more concentrated among a subset of individuals with other individuals experiencing little to no YLDs. Because of GBD COMO processes, our simulation should accurately estimate maternal disorder YLDs at the population level in the baseline scenario (in the absence of the above limitation) despite this limitation. Additionally, for not targeted interventions, this limitation should not substantially impact our estimation of intervention impact on maternal disorders cases and YLDs. However, for interventions that are targeted to high-risk groups, we will likely *underestimate* intervention impact as a result.s
```

Comment:
```suggestion
    - Our modeling strategy assumes that maternal disorders cases and YLDs are spread widely across the population of PLW rather than being more concentrated among a subset of individuals with other individuals experiencing little to no YLDs. Because of GBD COMO processes, our simulation should accurately estimate maternal disorder YLDs at the population level in the baseline scenario (in the absence of the above limitation) despite this limitation. Additionally, for not targeted interventions, this limitation should not substantially impact our estimation of intervention impact on maternal disorders cases and YLDs. However, for interventions that are targeted to high-risk groups, we will likely *underestimate* intervention impact as a result.
```